code/modules/mob/living/simple_animal/friendly/lizard.dm

```diff
@@ -24,13 +24,64 @@
 	can_collar = TRUE
 	mob_biotypes = MOB_ORGANIC | MOB_BEAST | MOB_REPTILE
 	gold_core_spawnable = FRIENDLY_SPAWN
+	var/eating_sound = 'sound/weapons/bite.ogg'
+	/// Lizards start with a tail
+	var/has_tail = TRUE 
+
+/mob/living/simple_animal/lizard/handle_automated_action()
+	if(!stat && !buckled)
+		if(prob(1))
+			custom_emote(EMOTE_VISIBLE, pick("sticks out its tongue", "wags its tail.", "lies down."))
+
+	if(!isturf(loc))
+		return
+	var/list/lizard_view = view(1, src)
+	for(var/mob/living/M in lizard_view)
+		if(M.stat == CONSCIOUS && Adjacent(M) && HAS_TRAIT(M, TRAIT_EDIBLE_BUG))
+			custom_emote(EMOTE_VISIBLE, "eats \the [M]!")
+			playsound(loc, eating_sound, 20, 1)
+			M.death()
```

Comment:
One last thing. Calling `death()` alone here might not suffice for some things. The butterfly has this problem right now because it doesn't have `del_on_death = TRUE` like the cockroach does. So it says the lizard *eats* the butterfly but the corpse sticks around.

I think the best thing might be to do something like:
```dm
M.death()
if(!QDESTROYING(src))
	qdel(src)
```

Reply:
I thought it was funny to keep it around, like it spit out it's wings